etcdserver/apply_v2.go

```diff
@@ -35,62 +35,69 @@ type applierV2 interface {
 	Sync(r *pb.Request) Response
 }
 
-type applierV2store struct{ s *EtcdServer }
+type ApplierV2store struct {
```

Comment:
can this stay private and instead have `func NewApplierV2Store(s *EtcdServer, c *membership.RaftCluster) ApplierV2`?